styles/pythonanywhere/catppuccin.user.css

```
@@ -0,0 +1,1417 @@
+          background-color: fade(@yellow, 20%);
+        }
+        .highlight-line-error {
+          background-color: fade(@red, 20%);
+        }
+        .ace_bracket {
+          border-color: @overlay1;
+        }
+      }
+
+      .ace_cursor {
+        color: @text;
+      }
+      .ace_marker-layer .ace_selection {
+        background: fade(@accent-color, 30%);
+      }
+      .ace_fold {
+        background-color: @surface0;
+        border-color: @surface1;
+      }
+
+      .ace_constant.ace_language,
+      .ace_keyword,
+      .ace_meta,
+      .ace_variable.ace_language {
+        color: @mauve;
+      }
+      .ace_line .ace_identifier:not(:first-of-type) {
+        color: @blue;
+      }
+      .ace_paren {
+        color: @red;
+      }
+      .ace_constant.ace_numeric {
+        color: @peach;
+      }
+      .ace_entity.ace_other.ace_attribute-name,
+      .ace_support.ace_constant,
+      .ace_support.ace_function {
+        color: @teal;
+      }
+      .ace_entity.ace_name.ace_tag,
+      .ace_support.ace_class,
+      .ace_support.ace_type {
+        color: @blue;
+      }
+      .ace_storage {
+        color: @peach;
+      }
+      .ace_string {
+        color: @green;
+      }
+      .ace_comment {
+        color: @overlay2;
+      }
+
+      .ace_indent-guide {
+        @svg: escape(
+          '<svg width="1" height="2" xmlns="http://www.w3.org/2000/svg"><path stroke="@{surface0}" d="M0 0v2"/></svg>'
+        );
+
+        background-image: url("data:image/svg+xml;utf8,@{svg}");
+      }
+    }
+
+    .ace_tooltip {
+      background-color: @crust;
+      color: @text;
+      border-color: @surface0;
+    }
+
+    #ace_settingsmenu {
+      background-color: @mantle;
+      box-shadow: none;
+      color: @subtext0;
+    }
+
+    #kbshortcutmenu {
+      background-color: @mantle;
+      box-shadow: none;
+
+      .ace_optionsMenuEntry {
+        &:hover,
+        &:focus {
+          background-color: @base;
+        }
+      }
+
+      &,
+      .ace_optionsMenuCommand {
+        color: @subtext0;
+      }
+
+      .ace_optionsMenuKey {
+        color: @accent-color;
+      }
+    }
+
+    /* Console */
+
+    x-screen {
+      background-color: @mantle !important;
+      color: @text !important;
+    }
+
+    .cursor-node {
+      border-color: @accent-color !important;
+      &[focus="true"] {
+        background-color: @accent-color !important;
+      }
+    }
+
+    /* $ prompt character */
+    [style*="color: rgb(0, 186, 19);"] {
+      color: @green !important;
+    }
+
+    /* ANSI colors */
+    x-row span {
+      /* Black & Bright Black */
+      &[style^="color: rgb(0, 0, 0);"],
+      &[style^="color: rgb(85, 87, 83);"] {
+        color: if(@lookup = latte, @subtext1, @crust) !important;
+      }
+      &[style*="background-color: rgb(0, 0, 0);"],
+      &[style*="background-color: rgb(85, 87, 83);"] {
+        background-color: if(@lookup = latte, @subtext1, @crust) !important;
+      }
```

```suggestion
      /* Black & Bright Black */
      &[style^="color: rgb(0, 0, 0);"],
      &[style^="color: rgb(85, 87, 83);"] {
        color: if(@lookup = latte, @subtext1, @mantle) !important;
      }
      &[style*="background-color: rgb(0, 0, 0);"],
      &[style*="background-color: rgb(85, 87, 83);"] {
        background-color: if(@lookup = latte, @subtext1, @mantle) !important;
      }
```
This makes the colors blend in better.

---

if we go off of https://github.com/catppuccin/palette/blob/main/palette.json, then shouldn't black and bright black be subtext1 and subtext0 (latte) or surface1 and surface2 (not latte)?

---

Sure feel free to change it.